src/pybind/mgr/dashboard/services/orchestrator.py

```diff
@@ -128,9 +128,10 @@ def reload(self, service_type, service_ids):
             raise_if_exception(c)
 
     @wait_api_result
-    def apply(self, service_spec: Dict) -> OrchResult[List[str]]:
+    def apply(self, service_spec: Dict, no_overwrite: Optional[bool]
+              = False) -> OrchResult[List[str]]:
```

Comment:
Isn't this a bit weird place to break a line in Python?

Reply:
Yup, updated accordingly